runtime/onert/core/src/ir/operation/BatchToSpaceND.cc

```diff
@@ -31,8 +31,8 @@ void BatchToSpaceND::accept(OperationVisitor &v) const { v.visit(*this); }
 
 BatchToSpaceND::BatchToSpaceND(const OperandIndexSequence &inputs,
                                const OperandIndexSequence &outputs)
-    : Operation{OperandConstraint::createExact(3u), inputs, outputs}
 {
+    : Operation{OperandConstraint::createExact(3u), inputs, outputs}
```

Comment:
"3u" mean input size?

Reply:
When doing setInputs, it checks if the number of inputs is correct.

```
void Operation::setInputs(const OperandIndexSequence &indexes)
{
  assert(_input_constr.check(indexes.size()));
  _inputs = indexes;
}
```